### DISCUSSION TITLE
How to launch modeller with 2D analysis type

### DISCUSSION THREAD

**Discussion by byronpiper** [ansys/pyansys-geometry#1185] [General]
I basically want to create a 2D aerofoil using PyGeometry but cannot seem to figure out if its possible to switch the analysis type from 3D to 2D. Is this functionality not supported?

**Comment by RobPasMue** [comment]
Hi @byronpiper  - thanks for opening this thread!

PyAnsys Geometry is intended for 3D modelling, just like SpaceClaim or Ansys Discovery are. It is a Python library for Ansys CAD products. We have the following example in which we create a 3D NACA airfoil: https://geometry.docs.pyansys.com/version/stable/examples/04_applied/01_naca_airfoils.html

Something you could do afterwards in Fluent is to perform a 2D analysis based on the 3D CAD model you just created (AFAIK).
